Spec/PushAdmin.swift

```diff
@@ -346,7 +330,7 @@ class PushAdmin : QuickSpec {
                             guard let device = device else {
                                 fail("Device is missing"); done(); return;
                             }
-                            expect(device).to(equal(self.deviceDetails))
+                            expect(device).to(equal(PushAdmin.deviceDetails))
                             expect(error).to(beNil())
```

Comment:
Here and below it can be `Self`?

Reply:
Thanks for teaching me something new! Fixed.